nevergrad/functions/gym/multigym.py

```diff
@@ -815,12 +815,15 @@ def action_cast(self, a):
             except AttributeError:
                 pass  # Sometimes an action space has no low and no high.
             if self.subaction_type is not None:
-                if type(a) == tuple:
-                    a = tuple(int(_a + 0.5) for _a in a)
+                if type(a) == tuple:      	
+                    try:
+                        a = tuple(int(_a + 0.5) for _a in a)
+                    except:  # oh my FSM! A tuple of nd-array probably.
+                        a = tuple(np.asarray(_a+.5, dtype=int) for _a in a)
```

Comment:
does it have to be tuples ? np.round(a) would be much simpler

Reply:
It depends on the gym problem, not on nevergrad.
The point is to deal with (at least some) nested structures which are not ndarray.

Reply:
usually I would expect it's simpler to convert everything to array, and then update the type if need be

Reply:
There are complicated types in Gym actually. You can have a dict of tuple of ndarray or stuff like that.